src/test/java/io/vertx/core/eventbus/LocalEventBusTest.java

```diff
@@ -41,7 +41,7 @@ public class LocalEventBusTest extends EventBusTestBase {
   private EventBusInternal eb;
   private boolean running;
 
-  public void setUp() throws Exception {
+  @Override public void setUp() throws Exception {
```

Comment:
Please configure the formatter of your IDE or text editor. The annotation should be on a separate line above.